ingest/src/main/java/com/microsoft/azure/kusto/ingest/source/TransformationMethod.java

```diff
@@ -0,0 +1,59 @@
+package com.microsoft.azure.kusto.ingest.source;
```

Comment:
this file should go one level up, in ingest/src/main/java/com/microsoft/azure/kusto/ingest